tests/unit/test_extras.cxx

```diff
@@ -50,3 +50,15 @@ ::testing::AssertionResult IsFieldPerpEqualBoutReal(const FieldPerp &field,
 
   return ::testing::AssertionSuccess();
 }
+
+/// Is \p field equal to \p number, with a tolerance of \p tolerance?
+::testing::AssertionResult IsBoutRealEqualBoutReal(BoutReal number1, BoutReal number2,
```

Comment:
Was `EXPECT_DOUBLE_EQ` too strict? Can you use `EXPECT_NEAR(val1, val2, abs_tol)` instead of adding this?

Reply:
Thank you, EXPECT_NEAR was what I wanted, just didn't know it existed. Will update...